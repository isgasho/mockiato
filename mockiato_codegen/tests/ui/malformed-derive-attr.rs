#![feature(plugin)]
#![plugin(mockiato_codegen)]

#[mockable(derive = "Foo")]
trait Foo {}
