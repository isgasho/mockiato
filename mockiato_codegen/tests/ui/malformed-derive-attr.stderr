error: #[mockable(derive(..)] must contain a list
 --> $DIR/malformed-derive-attr.rs:4:12
  |
4 | #[mockable(derive = "Foo")]
  |            ^^^^^^^^^^^^^^
  |
  = help: Example usage: #[mockable(derive(Eq, PartialEq))]

error: aborting due to previous error

